Sources/EngineKit/HashableExtensions.swift

```diff
@@ -46,229 +52,477 @@ extension Instructions: Hashable {
 			hasher.combine(bucketId)
 			hasher.combine(amount)
+
 		case let .createProofFromBucketOfNonFungibles(bucketId, ids):
 			hasher.combine("createProofFromBucketOfNonFungibles")
 			hasher.combine(bucketId)
 			hasher.combine(ids)
+
 		case let .createProofFromBucketOfAll(bucketId):
 			hasher.combine("createProofFromBucketOfAll")
 			hasher.combine(bucketId)
+
 		case let .burnResource(bucketId):
 			hasher.combine("burnResource")
 			hasher.combine(bucketId)
+
 		case let .cloneProof(proofId):
 			hasher.combine("cloneProof")
 			hasher.combine(proofId)
+
 		case let .dropProof(proofId):
 			hasher.combine("dropProof")
 			hasher.combine(proofId)
+
 		case let .callFunction(packageAddress, blueprintName, functionName, args):
 			hasher.combine("callFunction")
 			hasher.combine(packageAddress)
 			hasher.combine(blueprintName)
 			hasher.combine(functionName)
 			hasher.combine(args)
+
 		case let .callMethod(address, methodName, args):
 			hasher.combine("callMethod")
 			hasher.combine(address)
 			hasher.combine(methodName)
 			hasher.combine(args)
+
 		case let .callRoyaltyMethod(address, methodName, args):
 			hasher.combine("callRoyaltyMethod")
 			hasher.combine(address)
 			hasher.combine(methodName)
 			hasher.combine(args)
+
 		case let .callMetadataMethod(address, methodName, args):
 			hasher.combine("callMetadataMethod")
 			hasher.combine(address)
 			hasher.combine(methodName)
 			hasher.combine(args)
+
 		case let .assertWorktopContainsAny(resourceAddress):
 			hasher.combine("assertWorktopContainsAny")
 			hasher.combine(resourceAddress)
+
 		case let .callDirectVaultMethod(address, methodName, args):
 			hasher.combine("callDirectVaultMethod")
 			hasher.combine(address)
 			hasher.combine(methodName)
 			hasher.combine(args)
+
 		case let .allocateGlobalAddress(packageAddress, blueprintName):
 			hasher.combine("allocateGlobalAddress")
 			hasher.combine(packageAddress)
 			hasher.combine(blueprintName)
+
 		case .dropNamedProofs:
+			dummy(.dropNamedProofs)
 			hasher.combine("dropNamedProofs")
+
 		case .dropAuthZoneProofs:
+			dummy(.dropAuthZoneProofs)
 			hasher.combine("dropAuthZoneProofs")
+
 		case .dropAuthZoneRegularProofs:
+			dummy(.dropAuthZoneRegularProofs)
 			hasher.combine("dropAuthZoneRegularProofs")
+
 		case .dropAuthZoneSignatureProofs:
+			dummy(.dropAuthZoneSignatureProofs)
 			hasher.combine("dropAuthZoneSignatureProofs")
+
 		case let .callRoleAssignmentMethod(address, methodName, args):
 			hasher.combine("callRoleAssignmentMethod")
 			hasher.combine(address)
 			hasher.combine(methodName)
 			hasher.combine(args)
 		}
 	}
+}
 
-	public static func == (lhs: Instruction, rhs: Instruction) -> Bool {
-		switch (lhs, rhs) {
+// MARK: - Instruction + Equatable
+extension Instruction: Equatable {
+	public static func == (lhsOuter: Instruction, rhsOuter: Instruction) -> Bool {
+		// A function call like dummy(.someCase) will stop compiling if an
+		// associated value is later added to the pop case
```

Comment:
I guess this should say "added to `someCase`" now

Reply:
Remember that functions can have proper doc comments with /// and backticks for code